src/planwise/client/projects/handlers.cljs

```diff
@@ -81,5 +81,32 @@
          current-filter (get-in db path)
          toggled-filter (if (contains? current-filter filter-value)
                           (disj current-filter filter-value)
-                          (conj current-filter filter-value))]
-     (assoc-in db path toggled-filter))))
+                          (conj current-filter filter-value))
+         new-db (assoc-in db path toggled-filter)
+         updated-db-filters (get-in new-db [:facilities :filters])
+         updated-filters (into {} (for [[k v] updated-db-filters] [k (apply vector v)]))]
+     (api/fetch-facilities updated-filters :projects/facilities-loaded)
+     new-db)))
+
+(register-handler
+ :projects/facilities-loaded
+ in-current-project
+ (fn [db [_ response]]
+   (let [count-path [:facilities :count]
+         facilities-path [:facilities :facilities]
```

Comment:
Though it's a style choice, I'd avoid creating a binding for each of the paths, as they are quite simple and used only once